### PR TITLE
fixed readme typos js formatting issues

### PR DESCRIPTION
**Pull request by lamaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -425,21 +449,20 @@ Electron is employed for macOS, Windows and Linux apps.
...
 
-```
+```wh
 electron-packager . "SecureLogin" --osx-sign --overwrite --arch=x64 --icon=www/electron.icns
 electron-installer-dmg SecureLogin-darwin-x64/SecureLogin.app SecureLogin
 ```
 
-For MAS
-```
+For MAC
```
Comment (deltaidea) [inline README.md]: MAS == Mac App Store. See #15.

File: README.md [hunk 2]
```diff
@@ -152,21 +172,25 @@ The "Secure Login" button on your website/app opens the native SecureLogin app:
...
-      location.reload()
-    }else{
-      console.log(d)
+loginAccount.addEventListener('click', function() {
+  xhr(
+    '/login',
+    {
+      sltoken: SecureLogin(), //returns state and opens the app
+      authenticity_token: csrf
+    },
```
Comment (deltaidea) [inline README.md]: I'm all for generous spacing, but this is taking it too far IMO.
README should be concise.

File: README.md (hunk 1 above)
Comment (lamaster) [reply README.md]: Ok. That should be written explicit then, I guess. Not obvious at all.

File: README.md (hunk 2 above)
Comment (lamaster) [reply README.md]: I've just used prettier formatting guidelines.

**Review by deltaidea — APPROVED** [review]
👍 Thanks!

**Comment by homakov** [comment]
@lamaster thanks, protocol was changed dramatically and I just noticed this one so closing it... Btw what's with === vs ==?

**Comment by andrewda** [comment]
`===` is _generally_ better practice when just looking for one type. In this case it really doesn't matter, but in my code I almost always use triple equals unless there's a specific reason why I wouldn't.

**Pull request closed without merging**